src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -425,12 +425,13 @@ func runServerProxy(cfg *config.Config) {
 }
 
 func updateServerSideConfigClient(cfg *config.Config) {
-	client, err := util.HTTPClient(cfg.CloudConfigCA, "")
+	// temporarily comment it as we didn't have fronted dialer on server side
+	/*client, err := util.HTTPClient(cfg.CloudConfigCA, "")
```

Comment:
No we don't need it. As I changed `config.Configure` to accept `fronted.Dialer` as parameter, the call to it here becomes invalid. We can pass the `http.Client` as another parameter, but it makes code less understandable.

Extract those code to a common call like `nextHTTPClient` would be good to hide the difference.